src/help.js

```diff
@@ -0,0 +1,25 @@
+import shell from 'shelljs';
+
+const commandList = Object.keys(shell)
+  .filter(cmd => typeof shell[cmd] === 'function' && cmd !== 'ShellString');
```

Comment:
Ideally, this would be a part of shelljs itself.

Also, it would be nice if each command had a `.help` property which contained documentation.